src/actions.js

```diff
@@ -7,6 +7,10 @@ import {
   prepareMutation,
   graphqlWithVariables,
   fetchMutation,
+<<<<<<< Updated upstream
+=======
+  useGraphqlQuery,
+>>>>>>> Stashed changes
```

Comment:
```suggestion
 useGraphqlQuery,
```

that need to be done in other place where you have the merge signs
```
<<<<<<<<<< name of the branch 1
proposal from branch 1
==========
proposal from branch 2
>>>>>>>>>> name of the branch 2